### ISSUE TITLE
Fail to reproduce result of ResNet18 on ImageNet 

### ISSUE THREAD

**Issue by XiongweiWu** [pytorch/pytorch#4797]
Hi I tried the provided script in examples to train ImageNet but only got ~68% accuracy, which is far away from the expected result 69.76%. Can u give more suggestions? I add color augmentation in training step (transforms.ColorJitter(brightness=0.5, contrast=0.5, saturation=0.5, hue=0.5)) and other settings are identical to provided script.

**Comment by apaszke** [comment]
Note that we're using GitHub issues for bug reports only, and all questions should be posted on [our forums](discuss.pytorch.org).

**Comment by DeppMeng** [comment]
sorry but I can't open the link https://github.com/pytorch/pytorch/issues/discuss.pytorch.org
